### PR TITLE
Some FluentTab changes

### PR DESCRIPTION
**Pull request by vnbaaij**
Add a Visible property and control the id generation for the tab panel.

Also, some doc updates (not Tab related)

### CODE REVIEW

File: src/Core/Components/Tabs/FluentTab.razor.cs [hunk 1]
```diff
@@ -28,6 +28,18 @@ public partial class FluentTab : FluentComponentBase
...
+    /// Gets or sets the visability of a tab
+    /// </summary>
+    [Parameter]
+    public bool Visible { get; set; } = true;
+
+    /// <summary>
+    /// Callback to invoke when visibility changes
+    /// </summary>
+    [Parameter]
+    public EventCallback<bool> VisibleChanged { get; set; }
```
Comment (dvoituron) [inline src/Core/Components/Tabs/FluentTab.razor.cs:41]: Why to have a `VisibilityChanged`, if this event is never called by the component?

**Comment by dvoituron** [comment]
Did you check the behavior of the component with properties likes `ActiveTabId`  `ActiveTab` and `TabsOverflow`?

File: src/Core/Components/Tabs/FluentTab.razor.cs (hunk 1 above)
Comment (vnbaaij) [reply src/Core/Components/Tabs/FluentTab.razor.cs:41]: I was proactive and thought we would probably need it somewhere? 😂

**Comment by vnbaaij** [comment]
Yes, I did check. But lets hold off onmerging for a bit and let me  test some more things

File: src/Core/Components/Tabs/FluentTab.razor.cs (hunk 1 above)
Comment (dvoituron) [reply src/Core/Components/Tabs/FluentTab.razor.cs:41]: Mmmm ? No way 😀

**Review by dvoituron — APPROVED** [review]

**Pull request merged**